autoload/kite/codenav.vim

```diff
@@ -0,0 +1,38 @@
+function! kite#codenav#from_file()
+  call kite#codenav#request_related(expand('%:p'), v:null)
```

Comment:
In other places where we need an absolute path we use `kite#utils#filepath(0)`, which resolves symlinks.  I think it would be nice to keep things consistent.